apps/server/src/modules/school/service/school.service.spec.ts

```diff
@@ -190,6 +194,31 @@ describe('SchoolService', () => {
 		});
 	});
 
+	describe('getSchoolBySchoolNumber', () => {
```

Comment:
Please use this test structure https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/shared/domain/entity/submission.entity.spec.ts#L164